server/src/controllers/patient_controller.ts

```diff
@@ -56,6 +56,28 @@ export class PatientController implements interfaces.Controller {
             res.json(error.statusCode || 500, { error: error.message });
         }
     }
+
+    @Get("/:patientId/statuses/fields", "extractJwtMiddleware", "isValidPatientMiddleware")
+    private async getPatientStatusFields(req: Request, res: Response): Promise<void> {
+        try {
+            const { value, error } = Joi.object({ patientId: Joi.number().required() })
+                .required()
+                .validate({ patientId: req.params.patientId });
```

Comment:
Can you extract this to a variable like the other schemas just so they are consistent?